docs/tutorials/analysis/1D/spectral_analysis_hli.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>MAybe remove the comment in the <code>observations </code> section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/4044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>